code/obj/false_wall.dm

```diff
@@ -160,7 +159,7 @@ ADMIN_INTERACT_PROCS(/turf/simulated/wall/false_wall, proc/open, proc/close)
 		if (src.operating)
 			return 0
 		src.operating = 1
-		src.name = "false wall"
+		src.name = "false [src.material.name] wall"
```

Comment:
Can you rely on all false walls having a material? I'm guessing there might be some in azones and such that don't